yolog/configHandler.py

```diff
@@ -0,0 +1,21 @@
+from ConfigParser import SafeConfigParser
+import os, sys
+config = SafeConfigParser()
+path = '~/.yolog/config.ini'
+path = os.path.expandvars(os.path.expanduser(path))
+config.read(path)
```

Comment:
No leaky code please. 
This should go in `__init__` method of the class. (remove `pass` afterwards)

```python
def __init__(self, path):
    self.path = os.path.expandvars(os.path.expanduser(path))
    self.config = SafeConfigParser()
    self.config.read(self.path)
```